docs/conf.py

```diff
@@ -70,6 +70,11 @@
         "v{}/%s".format(STACVersion.DEFAULT_STAC_VERSION),
         "%s path",
     ),
+    "stac-api-spec": (
+        "https://github.com/radiantearth/stac-api-spec/tree/"
+        "v{}/%s".format(STACVersion.DEFAULT_STAC_VERSION),
```

Comment:
Is there a better way to get latest stac-api version? I assume latest version numbers won't always align?

Reply:
Correct ... I'd recommend just a new constant that we'd update when STAC API updates.